lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -20,12 +19,16 @@
             var userPreferencesModel = new UserPreferencesModel();
             userPreferencesModel.url = userPreferencesApiUrl;
 
-            var sectionsData = [
+            var aboutSectionsData = [
```

Comment:
Throughout this file there are multiple vars declared in the wrong place. Could you please update so all of the variables are declared at the top of the function?